src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -268,6 +275,9 @@ private void testSubmitAction() throws Exception {
 
         submitPage.submitWithoutConfirmationEmail();
 
+        confirmationEmailBox = submitPage.isConfirmationEmailBoxTicked();
+        assertFalse(confirmationEmailBox);
```

Comment:
These two lines can be combined to `assertTrue(submitPage.isConfirmationEmailBoxTicked());`